Rx/v2/src/rxcpp/rx-observable.hpp

```diff
@@ -2940,6 +2940,24 @@ class observable
         return  take_until(rxs::timer(when, cn), cn);
     }
 
+    /*! For the first items fulfilling the predicate from this observable emit them from the new observable that is returned.
+
+    \tparam Predicate  the type of the predicate
+
+    \param t  the predicate
+
+    \return  An observable that emits only the first items emitted by the source Observable fulfilling the predicate, or all of the items from the source observable if the predicate never returns false
+*/
```

Comment:
A snippet needs to be added to the docs in another PR.

Thanks!

example of the `take_until()` snippet [definition](https://github.com/Reactive-Extensions/RxCpp/blob/master/Rx/v2/examples/doxygen/take_until.cpp) & [build reference](https://github.com/Reactive-Extensions/RxCpp/blob/master/projects/doxygen/CMakeLists.txt#L102) & doc reference
```
/*
        \sample
        \snippet take_until.cpp threaded take_until time sample
        \snippet output.txt threaded take_until time sample
*/
```